app/views/roles/_organisations.html.erb

```diff
@@ -1,5 +1,5 @@
 <% if role.organisations %>
-  <div class="organisations-list">
+  <div class="organisations-list" dir="<%= page_text_direction %>" lang="<%= role.locale %>">
```

Comment:
This shouldn't be set `rtl`, since it contains a left-to-right language ('Organisations').